### PR TITLE
Getting Started section update

### PR DESCRIPTION
**Pull request by MightOfOaks**

### CODE REVIEW

File: docs/02-getting-started/01-intro.md [hunk 1]
```diff
@@ -5,51 +5,37 @@ id: intro
...
 
-This is designed for programmers who are comfortable with the command-line and using Linux or MacOS. It is helpful to
-have a basic understanding of Rust and Go, but we will lead you through, so you need no prior knowledge.
-The [next tutorial, name service](/tutorials/name-service/intro) will assume knowledge of these basics. And it will show
-all the development flow from zero to production.
+Once we are finished with setting up environment, deploying a smart contract to the testnet and interacting with it, we will be modifying the example Escrow Contract by adding a backdoor in the [Hijack Escrow tutorial](/tutorials/hijack-escrow/intro) in order to make things a bit more interesting. The modification will expose an identical API to the original one, except for the addition of a single hidden command. The idea behind making such a modification is twofold; one, familiarizing ourselves with what constitutes a smart contract, and the other is to manifest the importance of verifying the source code behind any contract you interact with.
```
Comment (orkunkl) [inline docs/02-getting-started/01-intro.md:19]: Hijack Escrow is an old implementation, maybe we can introduce a better next step like dev academy? This is not the scope of this PR though.

File: docs/02-getting-started/05-interact-with-contract.md [hunk 2]
```diff
@@ -2,102 +2,257 @@
 sidebar_position: 5
 ---
 
-# Uploading and Interacting
+# Deployment and Interaction
```
Comment (orkunkl) [inline docs/02-getting-started/05-interact-with-contract.md:5]: Can you check if all the code pieces mentioned below works before merging?

Comment (MightOfOaks) [reply docs/02-getting-started/05-interact-with-contract.md:5]: wasmd Go CLI example code does work as long as the environment variables are sourced properly. The user will be able to source the right variables during Environment Setup once the following PR is approved and merged.
https://github.com/CosmWasm/testnets/pull/151

Similarly, the CosmJS code depends on the following PR to be merged for generating wallet addresses.
https://github.com/InterWasm/cw-plus-helpers/pull/4

File: docs/02-getting-started/01-intro.md (hunk 1 above)
Comment (MightOfOaks) [reply docs/02-getting-started/01-intro.md:19]: Alright. Will make changes directing the user to the Dev Academy content with a new PR.

**Comment by orkunkl** [comment]
New changes has been added from a different PR, we need to resolve conflict before merging

**Pull request merged**